components/ActivityLogEvent.ts

```diff
@@ -43,26 +53,31 @@ export class ActivityLog {
     this.timestamp = timestamp;
     this.deviceName = deviceName;
     this.issuer = issuer;
+    this.credentialConfigurationId = credentialConfigurationId;
+    this.flow = flow;
   }
 
-  static logTamperedVCs() {
-    return {
-      _vcKey: '',
-      type: 'TAMPERED_VC_REMOVED',
-      timestamp: Date.now(),
-      deviceName: '',
-      vcLabel: '',
-      issuer: '',
-      id: '',
-      credentialConfigurationId: '',
-    };
+  getActionText(t: any, wellknown: Object | undefined) {
```

Comment:
instead of any t as any, can we specify the type

Reply:
changed it to type **TFunction**